src/util/xeval/evaluator.rs

```diff
@@ -513,6 +518,13 @@ impl Evaluator {
         Ok(Datum::Json(first.merge(suffixes)))
     }
 
+    fn eval_scalar_function(&mut self, ctx: &EvalContext, expr: &Expr) -> Result<Datum> {
+        match expr.get_sig() {
+            ScalarFuncSig::AbsInt => self.abs_int(ctx, expr),
```

Comment:
I think we should add all these "framework" stuff, such as protocol to function call mapping, function definition stubs, e.g. before letting the community join in. It's easier for anyone to just implement the stub function(which is `abs_int` here) without add other code.